android-components/plugins/publicsuffixlist/settings.gradle

```diff
@@ -2,6 +2,4 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-object AndroidComponents {
-    const val VERSION = "109.0.20221211143351"
-}
+// Prevents gradle builds from looking for a root settings.gradle
```

Comment:
Weird that this looks like the file moved. 🤔

Reply:
(In the diff view.)
<img width="1035" alt="Screenshot 2022-12-15 at 12 28 18 PM" src="https://user-images.githubusercontent.com/472523/207927903-33bdf632-0361-4653-a15e-d8c890c9f2ed.png">

Reply:
I know, but it should be fine.

Reply:
Yeah, let's land it.